obal/data/tasks/download_and_extract_source.yml

```diff
@@ -120,7 +124,6 @@
           {{ downstream_changelog }}
         insertafter: '^%changelog'
 
-    - include: tasks/setup_sources.yml
```

Comment:
This was no longer needed?

Reply:
Uh. Good catch. That file does not exist anymore. So it's correct to remove the line here. But that means `update_package.yml` is broken for downstream builds (and also was already before this PR).

Reply:
opened https://github.com/evgeni/obal/issues/24 to track this